Utilities/bootstrap

```diff
@@ -108,7 +108,7 @@ g_default_sysroot = None
 if platform.system() == 'Darwin':
     g_platform_path = subprocess.check_output(
```

Comment:
This should be renamed if you are going to reuse it for this purpose (I see now it is unused).

I would probably keep the platform path, and introduce a new path for `g_platform_developer_frameworks`.